crates/control/migrations/20220121215901_create_connectors.up.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE connectors (
+  "id" BIGINT NOT NULL DEFAULT id_generator(),
+  PRIMARY KEY (id),
+  "type" TEXT NOT NULL,
+  "name" TEXT NOT NULL,
+  "owner" TEXT NOT NULL,
```

Comment:
Right now `"owner"` is just a display name. That's how I was using it. I didn't want to add more than that right now, but I wanted to leave a placeholder for it.

As for docs, that's a reasonable point. I'm going to say we should lean on rustdoc, rather than sql comments, for most of this. If we've got a tricky thing in a sql file, it's probably worth leaving a comment about, but the rustdoc will be useful beyond/accessible the migration files. I'll swing back and add them in.

Reply:
I might suggest `maintainer` or `vendor` instead of `owner`, just as a nit pick, but I agree that it makes sense to have some sort of display name to tell users where the connector has come from.